control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -1604,6 +1604,50 @@ paths:
           $ref: '#/components/responses/ServerError'
       security:
         - JWT: [ ]
+    delete:
+      tags:
+        - Snapshots
+      operationId: delete_volume_snapshot
```

Comment:
Convention is using the http verb. So `del` rather than `delete`.
```suggestion
      operationId: del_volume_snapshot
```

Reply:
Right. Done.